frontend/public/style/coreos.css

```diff
@@ -290,14 +290,29 @@ input[type="radio"] {
     transform: scale(1, 1); } }
 
 .co-m-cog {
-  position: relative;
   color: #999;
   display: inline-block;
-  font-size: 14px; }
+  font-size: 14px;
+  position: relative; }
   .co-m-cog:hover {
     cursor: pointer; }
-  .co-m-cog.open .co-m-cog__icon {
+  .co-m-cog.open {
     color: #2276ad; }
+  .co-m-cog__icon {
+    color: $color-pf-black-700;
+  }
+
+.co-m-cog__button {
+  padding: 0 7px;
+  &:focus {
```

Comment:
You should be able to collapse these styles. Do we need active, too?

```sass
&:active,
&:focus,
&:hover {
  .co-m-cog__icon {
    color: $color-pf-blue-400;
  }
}
```

I suspect there is a link color variable we should use for the hover/active style, however.

Reply:
I don't see any variables that are pf-blue-400/#0088ce. Do you want me to create one?